performance-test/support-files/location_code_immediate_children.txt

```diff
@@ -0,0 +1,22 @@
+locationCodeImmediateChildren
+MOR
+RSK
```

Comment:
I see this values are repeated as in location code, what is the difference, can we not maintain it together ?

Reply:
Have made the location code Immediate Children part of the hierarchy level file.